java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -89,6 +91,10 @@ protected static String runCommand(String... command) {
             LOG.warning(String.format("%s running command %s: %s",
                     e.getClass().getSimpleName(), Arrays.toString(command), e.getMessage()));
         }
+        if (!output.startsWith(INFO)) {
+          throw new WebDriverException("Error running command: " + Arrays.toString(command));
```

Comment:
Maybe it deserves to include the value of `output` in the exception message, since that info can be relevant to discovered what happened.

Reply:
Hmm, at least in Ruby, it either has the driver info or it is an empty string. I'll see if that's a Ruby implementation detail and/or if Java is different.

Reply:
Yeah, if it fails it is either null because there was an exception, an empty string because console error doesn't return a value, or the version because somehow `--version` was passed in.